pages/help/help_domain-management.md

```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
 
 Manage DNS information, domain contacts, and more.
  
-- [Manage DNS information for your domain](#)
-- [Add people to your domain account](#)
-- [Add a security email for your domain](#)
-- [Renew your .gov domain](#)
-- [Domain security best practices](#)
+- [Manage DNS information for your domain](#manage-DNS-information-for-your-domain)
+- [Add or update the security email for your domain](#add-or-update-the-security-email-for-your-domain)
```

Comment:
This one isn't working, missing the `.gov` portion
`#add-or-update-the-security-email-for-your-.gov-domain`

Reply:
Fixed